.idea/externalDependencies.xml

```diff
@@ -2,6 +2,6 @@
 <project version="4">
   <component name="ExternalDependencies">
     <plugin id="com.dubreuia" />
-    <plugin id="google-java-format" min-version="1.12.0.0" />
+    <plugin id="google-java-format" min-version="1.15.0.0" />
```

Comment:
🔧 This just occurs to me now instead of in the previous PR. A comment here mentioning we need the extra .0 would be nice. I'm not sure if IntelliJ will override this though.

Reply:
This commit is already overridden with a new commit though, I doubt people would see this 😅 
My concern is that someone would just copy paste the version from the dependencies in here, not realising they need the extra 0.

Reply:
In my opinion, we should look at the history of a file to understand the reasoning behind previous changes. Also, the plugin might change its versioning scheme and then the comment in the config wouldn't fit anymore. I'm gonna leave it as it is. 😄  Hope you don't mind.

Reply:
It's a 🔧 for a reason 😄